src/Calculator.java

```diff
@@ -0,0 +1,28 @@
+public class Calculator {
+
+    public void rectangleArea(Rectangle rectangle) {
+        double x1 = rectangle.getP1().x;
+        double y1 = rectangle.getP1().y;
+        double x2 = rectangle.getP2().x;
+        double y2 = rectangle.getP2().y;
+        double x3 = rectangle.getP3().x;
+        double y3 = rectangle.getP3().y;
+
+        double a = ((x2 - x1) * (x2 - x1)) + ((y2 - y1) * (y2 - y1));
+        double sideA = Math.sqrt(a);
+
+        double b = ((x3 - x1) * (x3 - x1)) + ((y3 - y1) * (y3 - y1));
+        double sideB = Math.sqrt(b);
+
+        System.out.println(sideA * sideB);
```

Comment:
I would rather return rectangle area because it is calculator and should only calculate, not print.